extensions/positron-code-cells/src/extension.ts

```diff
@@ -6,18 +6,49 @@
 import * as vscode from 'vscode';
 import { initializeLogging } from './logging';
 import { CellCodeLensProvider } from './codeLenses';
-import { registerCommands } from './commands';
 import { activateDecorations } from './decorations';
 import { activateContextKeys } from './context';
+import { canHaveCells, destroyDocumentManager, getOrCreateDocumentManager } from './documentManager';
+import { registerCommands } from './commands';
 
-export const IGNORED_SCHEMES = ['vscode-notebook-cell', 'vscode-interactive-input'];
+export const IGNORED_SCHEMES = ['vscode-notebook-cell', 'vscode-interactive-input', 'inmemory'];
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
 	initializeLogging();
 
+	// When starting extension, fill documentManagers
+	const activeEditors = vscode.window.visibleTextEditors;
+	activeEditors.forEach((editor) => {
+		if (canHaveCells(editor.document)) {
+			const docManager = getOrCreateDocumentManager(editor.document);
+			docManager.parseCells();
```

Comment:
We could make a minor improvement here and only parse cells for the set of open _documents_ not editors. I think the first `getOrCreateDocumentManager` call for a document will also call `parseCells` so maybe that isn't needed too.